sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -459,6 +460,29 @@ object SQLConf {
     .intConf
     .createWithDefault(4096)
 
+  val IS_PARQUET_PARTITION_ADAPTIVE_ENABLED = buildConf("spark.sql.parquet.adaptiveFileSplit")
+    .doc("For columnar file format (e.g., Parquet), it's possible that only few (not all) " +
+      "columns are needed. So, it's better to make sure that the total size of the selected " +
+      "columns is about 128 MB "
```

Comment:
It sounds not describing what the configuration does actually.

Reply:
I updated the description just now. Please help to review it again. Thanks a lot @HyukjinKwon